ldk/javascript/src/whisper/form.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Components, Form, Whisper, WhisperComponentType } from '.';
+
+export class LdkForm {
+    children: Array<Components>;
+
+    componentState: Map<string, any>;
+
+    constructor(children: Array<Components>) {
+        this.componentState = new Map<string, any>();
+        this.children = children;
+
+        this.setInitialComponentState(this.children);
+        this.registerListeners(this.children);
+    }
+
+    getComponentState(): Map<string, any> {
+        return this.componentState;
+    }
+
+    private isOnChangeInput(child: any): boolean {
+        return (child.onChange && child.name);
+    }
+
+    private isOnSelectInput(child: any): boolean {
+        return (child.onSelect && child.name);
+    }
+
+    private setInitialComponentState(children: Array<any>) {
+        children.forEach((child: any) => {
+            if (this.isOnChangeInput(child)) {
+                this.componentState = this.componentState.set(child.name, child.value);
```

Comment:
The `Map` object here is being mutated when `Map.set` is called. Shouldn't need to set the `componentState` property again.

Reply:
Removed double assignment.